docs/install/containerd/containerd-install-guide.md

```diff
@@ -0,0 +1,99 @@
+# Install Kata Containers with containerd
+
+1. Perform initial checks
+
+   Check to see if Kata Containers is already installed:
+
+   ```bash
+   $ command -v kata-runtime &>/dev/null && { echo >&2 "ERROR: Please remove existing Kata Containers installation"; exit 1; }
+   ```
+
+1. Install dependencies
+
+   ```bash
+   $ source /etc/os-release || source /usr/lib/os-release
+   $ packages='containerd curl git jq'
+   $ case "$ID" in
+   $   debian|ubuntu) sudo apt-get -y install $packages ;;
+   $   *) echo >&2 "ERROR: Unsupported distro: $ID"; exit 1 ;;
+   $ esac
+   ```
+
+1. Install Kata Containers
+
+   ```bash
+   $ tmpdir=$(mktemp -d)
+   $ pushd "$tmpdir" &>/dev/null
+   $ url="https://api.github.com/repos/kata-containers/runtime/releases/latest"
+   $ full_url=$(curl -sL "$url" | jq -r '.assets[0].browser_download_url')
+   $ arch=$(uname -m)
+   $ echo "$full_url" | grep -q "$arch" || { echo >&2 "ERROR: No Kata Containers package for $arch architecture"; exit 1; }
+   $ curl -LO "$full_url"
+   $ file=$(echo "$full_url" | awk -F\/ '{print $NF}')
+   $ sudo tar -C / -xvf "./${file}"
+   $ popd &>/dev/null
+   ```
+
+   Show version details:
+
+   ```bash
+   $ export PATH=$PATH:/opt/kata/bin
+   $ kata-runtime --version
+   ```
+
+1. Configure containerd
+
+   Stop the service to allow the configuration to be modified:
+
+   ```bash
+   $ sudo systemctl stop containerd
+   ```
+
+   Backup the original containerd configuration:
+
+   ```bash
+   $ cfg="/etc/containerd/config.toml"
+   $ sudo grep -q "io.containerd.kata.v2" "$cfg" || sudo cp "$cfg" "${cfg}.ORIGINAL"
+   ```
+
+   Add the Kata Containers configuration details:
+
+   ```bash
+   $ sudo grep -q "io.containerd.kata.v2" "$cfg" || cat <<EOT | sudo tee -a "$cfg"
+   $ [plugins]
```

Comment:
`$`s in config content should be removed?

Reply:
Although the script that converts the doc to a script copes with this, I agree - this looks incorrect as shell doesn't show the same prompt for here docs. Fixed.